templates/lune/partials/product/product.snip.html

```diff
@@ -0,0 +1,175 @@
+<div class="lune-page lune-page-product {{classes}}">
+  <div class="md-col-9 mx-auto {{classes}}">
+
+    {{#hero}}
```

Comment:
I think this partial  lacks a semantic structure 

https://www.iandevlin.com/blog/2011/04/html5/html5-section-or-article
```html
<article>
  <section>
      <header>If needed - this can contain headings images or anything that is a header to that section</header>
      <div  class="content"></div>
      <footer>If needed - action bars , cta etc - whatever constitutes the footer of the section</footer>
   </section>

 <section>
  Though not ideal i can also just contain only content 
 </section>
</article>

```